xbmc/platform/linux/input/LibInputKeyboard.cpp

```diff
@@ -28,159 +31,266 @@
 
 namespace
 {
-constexpr int REPEAT_DELAY = 400;
-constexpr int REPEAT_RATE = 80;
+constexpr unsigned int REPEAT_DELAY = 400;
+constexpr unsigned int REPEAT_RATE = 80;
+
+static const std::unordered_map<xkb_keysym_t, XBMCKey> xkbMap = {
```

Comment:
We have a constexpr map class you could use in utils/Map.h

The benefit is that it's possible to add a compile time static_assert to check if the map changes.

Reply:
Hope I did this correctly. I didn't add the static_assert though.